Documentation/building/freebsd-instructions.md

```diff
@@ -22,11 +22,12 @@ Install the following packages for the toolchain:
 - llvm37 (includes LLVM 3.7, Clang 3.7 and LLDB 3.7)
 - libunwind
 - gettext
+- icu
 
 To install the packages you need:
 
 ```sh
-janhenke@freebsd-frankfurt:~ % sudo pkg install bash cmake libunwind gettext llvm37
+janhenke@freebsd-frankfurt:~ % sudo pkg install bash cmake libunwind gettext llvm37 icu
```

Comment:
Not related to your changes, but do we actually want people's user/machine names in our general instructions?